tools/validators/instance_validator/tests/ontology_validation_test.py

```diff
@@ -0,0 +1,113 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for ontology_validation.py"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from universe_generation import generate_universe
+from universe_generation import ontology_validation
+from absl.testing import absltest
+
+_GOOD_EXAMPLE = {'UK-LON-S2': {'type': 'FACILITIES/BUILDING',
```

Comment:
can we use the yaml files that are already on github?

Reply:
Because of the way I've set up the PRs, I'll have to use the YAML files later; I need the syntax code to parse them.